userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1401,10 +1446,8 @@ int32_t scap_bpf_load(
 		return SCAP_FAILURE;
 	}
 
-	/* Store interesting Tracepoints */
-	memcpy(&handle->open_tp_set, &oargs->tp_of_interest, sizeof(interesting_tp_set));
-	/* Start with all tracepoints disabled */
-	interesting_tp_set initial_tp_set = {0};
+	/* Start with all tp disabled */
+	interesting_ppm_tp_set initial_tp_set = {0};
```

Comment:
I haven't looked at the code but does load_tracepoints do anything with an empty set?